remix-debug/README.md

```diff
@@ -16,15 +16,18 @@ npm install remix-debug
 
 ```javascript
 var Debugger = require('remix-debug').EthDebugger
-var BreakpointManager = require('remix-debug').EthDebugger
+var BreakpointManager = require('remix-debug').BreakpointManager
 
 var debugger = new Debugger({
   compilationResult: () => {
     return compilationResult // that helps resolving source location
   }
 })
 
-debugger.addProvider(web3, 'web3')
+// Sample http provider, but your url or provider may be different.
+var web3Provider = new Web3.providers.HttpProvider("http://localhost:7545")
+
+debugger.addProvider('web3', web3Provider)
```

Comment:
The order was wrong yes, but the type should be type of web3 object, not really `Web3.providers.<provider>`.
(I agree that the method name is a bit confusing)